.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: false
+language: go
+go:
+- 1.13.x
+
+before_install:
+- cd ../../../
```

Comment:
On my scale of things I am not proud of this is 6 out of 10 but I don't really know how to do it better (alternative is to force everyone to clone it to directory $GOPATH/github.com/Virtuslab )

Reply:
Isn't some env var available to refer to that location more explicitly?

Reply:
I'm not saying Travis offers such thing, I'm just curious if it is really a standard way of doing that.

Reply:
No, I know now. 

The cause is that Golang requires you to put your project inside dir: $GOPATH/github.com/{repo_org}/{repo_name}

so if you just put it inside:
$GOPATH/{repo_name} 

(which is default when you clone repo) it will not work. This is by design in golang. Unfortunately, people need to exactly know where to clone repo which makes me sad. 

This is supposed to change in Golang modules alas they are yet in beta :/